### DISCUSSION TITLE
개발자의 학습과 성장 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#25] [Announcements]
# 개발자의 학습과 성장 | kciter.so

🚨 이 자료는 개발자의 소프트 스킬이 아닌 하드 스킬을 이야기합니다. 또한, 성장을 강요하는 자료도 아닙니다. 이 자료는 개발자가 성장하기 위한 방법론을 이야기합니다. 기술을 다루는 다른 직업과 마찬가지로 개발자 또한  끊임없는 학습 을 해야한다. …

https://kciter.so/posts/developers-learning-and-growth

**Comment by ibcylon** [comment]
후기가 기다려진다

**Comment by Songhyun9** [comment]
너무 좋은 글이네요 잘 읽었습니다 ^^

**Comment by JaeSeoKim** [comment]
너무 좋은 글 입니다..! 덕분에 많은걸 다시 깨닫게 되는 계기가 되었어요.! 감사합니다.!!!

**Comment by ChoiInYeol** [comment]
10번이고 읽고 싶은 글이네요 감사합니다 ^^

**Comment by jinseriouspark** [comment]
정말 잘 봤습니다! 코드 따라치기 이야기 해주셨는데, 이 글이 너무 좋아서 글도 따라쳤어요 ㅎㅎㅎㅎ 
앞으로 더 많은 글 기다리고 있겠습니다

**Comment by sewonkimm** [comment]
안녕하세요! 글또 큐레이션을 통해서 글을 보게 되었는데 읽다보니 재작년에 제가 팀원분들께 공유해드린 발표자료의 주인이셨었네요(반갑) 좋은 글은 다시 한 번 봐도 좋네요~ 주기적으로 보겠습니다 👍

**Comment by LEEJIYUL** [comment]
안녕하세요! 좋은 글 감사합니다! 개발자 성장에 대한 생각에 좋은 지침이 되었습니다. 
이번에 개발자 성장에 관한 발표 자료를 만들려고하는데, 혹시 작성자님의 글을 인용해서 사용해도 될까요? 
물론 출저를 남기는 등 인용 규칙에 따라 인용 할 생각입니다!

**Reply by kciter** [reply]
안녕하세요 :) 편하게 사용하셔도 괜찮습니다!
